lib/auth/auth_with_roles.go

```diff
@@ -1263,6 +1263,11 @@ func (c *resourceAccess) checkAccess(resource types.ResourceWithLabels, filter s
 		return false, nil
 	}
 
+	// KindSAMLIdPServiceProvider does not support label matcher
```

Comment:
Let's leave a todo item here to re-enable this when we're ready to properly introduce role v8? And maybe link this PR as a reference.